.github/workflows/auto-rebase-open-prs.yml

```
@@ -0,0 +1,77 @@
+                  if git push --force-with-lease origin "$head"; then
+                      gh pr edit "$num" --remove-label merge-conflict 2>/dev/null || true
+                  else
+                      echo "  push rejected (concurrent update); will retry on next main push"
+                      git rebase --abort 2>/dev/null || true
+                  fi
+              else
+                  echo "  rebase has conflicts, flag PR"
+                  git rebase --abort
+                  gh pr edit "$num" --add-label merge-conflict 2>/dev/null || true
+                  gh pr comment "$num" --body \
+"Auto-rebase onto main failed: conflicts. Resolve locally:
```

actionlint (1.7.12)

[error] could not parse as YAML: found unknown escape character

(syntax-check)